index.js

```diff
@@ -167,10 +168,101 @@ module.exports = ({ app }) => {
   }
 }
 
-function handleOneCommit (pr, dcoFailed) {
-  return `You only have one commit incorrectly signed off! To fix, first ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally). Next, head to your local branch and run: \n\`\`\`bash\ngit commit --amend --no-edit --signoff\n\`\`\`\nNow your commits will have your sign off. Next run \n\`\`\`bash\ngit push --force-with-lease origin ${pr.head.ref}\n\`\`\``
-}
+function handleCommits (pr, commitLength, dcoFailed, allowRemediationCommits) {
+  let returnMessage = ''
+
+  if (dcoFailed.length === 1) {
+    returnMessage = 'There is one commit incorrectly signed off.  This means that the author of this commit failed to include a Signed-off-by line in the commit message.\n\n'
+  } else {
+    returnMessage = `There are ${dcoFailed.length} commits incorrectly signed off.  This means that the author(s) of these commits failed to include a Signed-off-by line in their commit message.\n\n`
+  }
+
+  returnMessage = returnMessage + `To avoid having PRs blocked in the future, always include \`Signed-off-by: Author Name <authoremail@example.com>\` in *every* commit message. You can also do this automatically by using the -s flag (i.e., \`git commit -s\`).
+
+Here is how to fix the problem so that this code can be merged.\n\n`
+
+  let rebaseWarning = ''
+
+  if (allowRemediationCommits.individual || allowRemediationCommits.thirdparty) {
```

Comment:
```suggestion
  if (allowRemediationCommits.individual || allowRemediationCommits.thirdParty) {
```

Reply:
I caught this as I was adding more tests. No need to change the current PR, let's merge this into a local branch (not `master`) then I'll send a PR with my changes against it